Rakefile

```diff
@@ -23,8 +23,10 @@ end
 
 task :build => :verify_private_key_present
 
-require 'rubocop/rake_task'
-desc 'Run RuboCop on the lib directory'
-RuboCop::RakeTask.new(:rubocop) do |task|
-  task.patterns = ['lib/**/*.rb']
+if RUBY_VERSION >= '2.4' && RUBY_ENGINE == 'ruby'
+  require 'rubocop/rake_task'
```

Comment:
It's preferable to wrap this in a rescue block instead:
e.g.
```
begin
  ... 
rescue LoadError
  # No rubocop means no rubocop rake task
end
```

Reply:
I've updated to rescue the `LoadError`.